.github/workflows/test-application.yaml

```diff
@@ -8,6 +8,9 @@ on:
             - '[0-9]+.[0-9]+'
     workflow_dispatch:
 
+env:
+    IS_GITHUB_ACTION: 1
```

Comment:
would more go with explicit name like `CI_GROUP_OUTPUT=1`

else we would not require to define an own variable as I think github would always set `GITHUB_ACTIONS=true`: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables 

I'm fine with both changes.